doc/answerfile.txt

```diff
@@ -256,8 +256,8 @@ Common Elements
 (Re)Install Attributes
 ----------------------
 
-  <installation sr-type="lvm|ext"?>
-  <installation srtype="lvm|ext"?>[D]
+  <installation sr-type="lvm|ext">?
```

Comment:
I think the interrogation mark here was really meant to be next to the `sr-type` parameter, rather than next to the whole XML tag.

Reply:
right, dropping that line